frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -96,6 +97,18 @@ export const Receive: FunctionComponent<Props> = ({
     }
   }, [addressType, availableScriptTypes, receiveAddresses]);
 
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    if (addressDialog) {
+      e.preventDefault();
+      setActiveIndex(0);
+      setAddressType(addressDialog.addressType);
+      dialogRef.current.deactivate();
+      setTimeout(() => {
+        setAddressDialog(undefined);
+      }, 350);
+    }
+  };
+
```

Comment:
`setTimeout` and `deactivate()` needed for the same reasons [above](https://github.com/digitalbitbox/bitbox-wallet-app/pull/1842/files#r1022541054).